File: webapi/Models/Views/EmployeeView.cs
namespace webapi.Models.Views
{
    public record EmployeeView
    {
        public long EmployeeId { get; set; }
        public required string EmployeeName { get; set; }
        public required string Email { get; set; }
        public required string PhoneNumber { get; set; }
        public string? ManagerName { get; set; }
        public required string DepartmentName { get; set; }
        public required string PositionName { get; set; }
        public long PersonalDataId {get; set;}
        public long? ContractId {get; set;}
    }
}

File: reactapp/src/pages/Employees/models/EmployeeView.ts
import { Optional } from "@/types/Optional";

//select type
export interface EmployeeView {
    employeeId: int,
    employeeName: string;
    workEmail: string;
    phoneNumber: string;
    managerName: Optional<string>;
    departmentName: Optional<string>;
    positionName: Optional<string>;
    insuranceTypeCode: Optional<string>,
    personalDataId: int,
    endMonthStatus: boolean
}